tests/macros/InterfaceOverview.test.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Test the API Interface Overview page content macro
+ *
+ * @prettier
+ */
+
+// Get necessary modules
+const {
+    assert,
+    describeMacro,
+    beforeEachMacro,
+    itMacro,
+    readJSONFixture
+} = require('./utils');
+const { JSDOM } = require('jsdom');
+
+const INTERFACE_DATA = JSON.stringify([
+    {
+        XMLHttpRequestEventTarget: {
+            inh: 'EventTarget',
+            impl: []
+        }
+    }
+]);
+
+const DATA = ((...dataNames) => {
+    /** @type {Record<string, any>} */
+    let result = {};
+    for (const name of dataNames) {
+        result[name] = readJSONFixture('interface-overview', name);
+    }
+    return result;
+})('EventTarget', 'XMLHttpRequestEventTarget');
+
+/**
+ * @param {DocumentFragment} dom
+ * @param {string[]} headers
+ */
+function checkSectionTitles(dom, headers) {
+    // An unclosed <dl> would just wrap all the remaining <h2>s,
+    // which would no longer be directly accessible through
+    // the `dom.children` property, as they’re no longer
+    // direct descendants.
+    let children = Array.from(dom.children).filter(n => n.matches('h2'));
+    expect(children.map(h => h.textContent.trim())).toEqual(headers);
+}
+
+describeMacro('InterfaceOverview', () => {
+    beforeEachMacro(macro => {
+        macro.mockTemplate('GroupData', INTERFACE_DATA);
```

Comment:
If you do end up deleting the group-related code in the macro, then you should also delete this line and the definition of INTERFACE_DATA above, I think.

Reply:
I accidentally wrote `GroupData` instead of `InterfaceData`.